### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cldougl**
add chart images to readme

@etpinard is that ok there or would you prefer to add the image to the doc section?

### CODE REVIEW

**Comment by etpinard** [comment]
Looks good! @cldougl can you wrap the image in an anchor so that click on it lead you to https://plot.ly/javascript/ ?

**Comment by etpinard** [comment]
... or to some other page that might be better.

**Comment by cldougl** [comment]
yep - I think https://plot.ly/javascript/ will be best for now

**Comment by rreusser** [comment]
Also, I think it's pretty safe to swap the markdown image for `` (I'm pretty sure that's the attr) in order to center it.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ plotly.js is a high-level, declarative charting library. plotly.js ships with ov
 chart types, including scientific charts, 3D graphs, statistical charts, SVG maps, financial charts,
 and more.
 
-![](https://raw.githubusercontent.com/cldougl/plot_images/add_r_img/plotly_2017.png)
+<p align="center">
+    <a href="https://www.plot.ly/javascript">
```
Comment (etpinard) [inline README.md]: maybe a `target="_blank"` here?

Comment (cldougl) [inline README.md]: oh yes, good call

**Comment by rreusser** [comment]
(FYI: I use [vmd](https://github.com/yoshuawuyts/vmd) all the time for working on + previewing github markdown files locally. Auto-updates on save.)

**Comment by etpinard** [comment]
:dancer:

**Pull request merged**